### ISSUE TITLE
Controls

### ISSUE THREAD

**Issue by jamesandrews23** [jamesandrews23/rpg-game#4]
Something we'll have to solve early is how are we going to set up controls? There are lots of options with phaser, my personal favorite is using a controller, which phaser supports, but there are other ways like virtual controls, or even just keyboard controls. I'm thinking we'll have to support more than one set of controls, but for now we could set up keyboard and later incorporate gamepads and other devices.

**Comment by jamesandrews23** [comment]
I've been thinking about what kind of controls we should incorporate into the game and I'm thinking that maybe we should start with the keyboard and mouse devices then later add game controller.

**Comment by jpaugh** [comment]
Good idea. Could we possibly do this in a generic way? That is, supporting a controller with graceful fallback to keyboard and mouse? If we could support all 3 independently (e.g. it's possible to use only the mouse, or only the keyboard), that would be ideal. But that might be a bit much to do right away.
